### ISSUE TITLE
Put micromamba binary somewhere else?

### ISSUE THREAD

**Issue by wolfv** [mamba-org/provision-with-micromamba#13]
I think currently we download the micromamba binary right there. Unfortunately on Windows, that means calling "micromamba" might accidentally invoke the exe directly (and not through the shell functions ... ) 

So a first step that would be good would be to move the micromamba binary out of current working directory.

**Comment by beckermr** [comment]
As long as it remains on the path it doesn't matter where it goes.

**Comment by wolfv** [comment]
for `activate` to work properly it should.
However, I can imagine a situation where someone is just using `bash` instead of `bash -l {0}` and thus might not have the shell properly set up. 
`micromamba install ...` would still work properly if it's in `PATH` then, though.

Although we might also have problems putting custom elemetns on `PATH` if we can't modify `.bashrc`. 

So we could move it to a global "bin" directory such as `/usr/bin/...` or so.

Idk if it's worth it.

**Comment by adriendelsalle** [comment]
Yeah maybe it would be quite magic to get micromamba capable to install packages but hard to understand why activation doesn't
